subt/config/r2.json

```diff
@@ -110,7 +110,7 @@
       },
       "radio": {
           "driver": "subt.radio:Radio",
-          "in": ["radio", "pose2d", "artf"],
+          "in": ["radio", "pose3d", "artf", "sim_time_sec"],
```

Comment:
Each and every time I comment that these changes are pointless and the lines should be removed instead.